source/reference/minio-server/settings/object-lambda.rst

```diff
@@ -10,49 +10,193 @@ Object Lambda Function Settings
    :local:
    :depth: 2
 
-This page documents environment variables for configuring MinIO to publish data to an HTTP webhook endpoint and trigger an Object Lambda function.
-See :ref:`developers-object-lambda` for more complete documentation and tutorials on using these environment variables.
+This page documents settings for configuring MinIO to publish data to an HTTP webhook endpoint and trigger an Object Lambda function.
+See :ref:`developers-object-lambda` for more complete documentation and tutorials on using these settings.
 
-You can specify multiple webhook endpoints as Lambda targets by appending a unique identifier ``_FUNCTIONNAME`` for each Object Lambda function.
-For example, the following command sets two distinct Object Lambda webhook endpoints:
+You can establish or modify settings by defining:
 
-.. code-block:: shell
-   :class: copyable
+- an *environment variable* on the host system prior to starting or restarting the MinIO Server.
+  Refer to your operating system's documentation for how to define an environment variable.
+- a *configuration setting* using :mc:`mc admin config set`.
 
-   export MINIO_LAMBDA_WEBHOOK_ENABLE_myfunction="on"
-   export MINIO_LAMBDA_WEBHOOK_ENDPOINT_myfunction="http://webhook-1.example.net"
-   export MINIO_LAMBDA_WEBHOOK_ENABLE_yourfunction="on"
-   export MINIO_LAMBDA_WEBHOOK_ENDPOINT_yourfunction="http://webhook-2.example.net"
+If you define both an environment variable and the similar configuration setting, MinIO uses the environment variable value.
+
+Some settings have only an environment variable or a configuration setting, but not both.
 
 .. include:: /includes/common-mc-admin-config.rst
    :start-after: start-minio-settings-test-before-prod
    :end-before: end-minio-settings-test-before-prod
 
-Environment Variables
----------------------
 
-.. envvar:: MINIO_LAMBDA_WEBHOOK_ENABLE
+Enable
+------
+
+.. tab-set::
+
+   .. tab-item:: Environment Variable
+      :sync: envvar
+
+      .. envvar:: MINIO_LAMBDA_WEBHOOK_ENABLE
+
+         Specify ``"on"`` to enable the Object Lambda webhook endpoint for a handler function.
+
+         Requires specifying :envvar:`MINIO_LAMBDA_WEBHOOK_ENDPOINT`.
+
+	 You can specify multiple webhooks as Lambda targets by appending a unique identifier ``_FUNCTIONNAME`` for each Object Lambda function.
+         For example, the following command enables two distinct Object Lambda webhook endpoints:
+
+         .. code-block:: shell
+            :class: copyable
+
+            export MINIO_LAMBDA_WEBHOOK_ENABLE_myfunction="on"
+            export MINIO_LAMBDA_WEBHOOK_ENABLE_yourfunction="on"
+
+   .. tab-item:: Configuration Setting
+      :sync: config
+
+      .. mc-conf:: lambda_webhook enable
+         :delimiter: " "
+
+         *Optional*
+
+         Specify ``"on"`` to enable the Object Lambda webhook endpoint for a handler function.
+	 Requires specifying :mc-conf:`~lambda_webhook.endpoint`.
```

Comment:
```suggestion
	 Requires specifying a :mc-conf:`~lambda_webhook.endpoint`.
```

A definite or indefinite article here?

Reply:
🤔 this mirrors usage elsewhere where one setting requires another. Is it something we should consider changing more widely?

Reply:
ah, no, leave it